edge-modules/Relayer/Settings.cs

```diff
@@ -27,17 +27,23 @@ public class Settings
                 return new Settings(
                     configuration.GetValue("transportType", TransportType.Amqp_Tcp_Only),
                     configuration.GetValue("inputName", "input1"),
-                    configuration.GetValue("outputName", "output1"));
+                    configuration.GetValue("outputName", "output1"),
+                    configuration.GetValue<Uri>("testResultCoordinatorUrl", new Uri("http://testresultcoordinator:5001")),
+                    configuration.GetValue<string>("IOTEDGE_MODULEID"));
```

Comment:
default value is [http://testresultcoordinator:5001/api/testoperationresult](http://testresultcoordinator:5001/api/testoperationresult)

Reply:
Good catch 👍